script.js

```diff
@@ -6,4 +9,317 @@ function openNav(){
+    $('#elementPoints').keypress(function(e){
+        if (e.keyCode == 13)
+        {
+            $('#scoringSaveButton').click();
+        }
+    });
+
+    //adds new element to table, handles things like element already exists and season not checked
+    $("#scoringSaveButton").click(function(){
+        var newElementName = $("#elementKey").val().trim().toString();
+        var newElementPoints = $("#elementPoints").val().trim();
+        
+        if (newElementName !== "" || newElementPoints !== "")
+        {
+            var season = $("input[name=seasonListItem]:checked","#seasonRadioList").val();
+            if (typeof(season) !== "undefined")
+            {
+                var cookieName = '/season_config/' + season + '/' + newElementName;
+                var cookieValue = newElementPoints;
+                
+                var cookieList = document.cookie.split(";");
+                var elementAlreadyStored = false;
+                for (var i = 0; i < cookieList.length;i++)
+                {
+                    if (cookieList[i].trim().startsWith('/season_config/' + season))
+                    {
+                        var str = cookieList[i];
+                        var checkerName = str.substring(str.lastIndexOf('/')+1,str.indexOf('='));
+                        if (checkerName == newElementName)
+                        {
+                            elementAlreadyStored = true;
+                            break;
+                        }
+                    }
+                }
+                
+                if (!elementAlreadyStored)
+                {
+                    $("#elementConfirmationBox").text(" ");
+                    document.cookie = cookieName + "=" + cookieValue + 
+                    "; expires=Wed, 1 Jan 2038 12:00:00 UTC; path=/";
+                    //doc.cookie --> /season_config/season/elementName=elementValue;...
+
+                    var newTRHtml = "<tr><td>" + newElementName + 
+                    "</td><td>" + newElementPoints + "</td></tr>";
+                    
+                    $("#scoringTable").append(newTRHtml);
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" added.<\i>');
+
+                    $("#elementKey").val("");
+                    $("#elementPoints").val("");
+                } else
+                {
+                    $("#elementConfirmationBox").html('<i>Element "' + newElementName + '" already exists.<\i>');
+                }
+            } else
+            {
+                $("#elementConfirmationBox").html("<i>Please check a season.<\i>");
+            }
+        } else
+        {
+            $('#elementConfirmationBox').html("<i>Please fill in both fields.<\i>");
+        }      
+    });
+
+
+    //start of questionnaire js code
+    $("#pitScoutingSubmit").click(function(){
+        var allQuestions = $("#pitScoutingTextBox").val().split('\n');
```

Comment:
So, this will split on all OS's, but on Windows, at '\r' will be on the tail of each line. Are you trimming them? Otherwise, use the regex split I sent you to grab the '\r' as well if present.